Client/MirScenes/GameScene.cs

```diff
@@ -8430,6 +8437,29 @@ public void PlaySound(S.PlaySound p)
             SoundManager.PlaySound(p.Sound, false);
         }
 
+        public void ShowLogNotice(S.UpdateLogNotice p)
+        {
+
+            string[] stringSeps = new string[] { "\r\n" };
+            LoginNoticeDialog.notice = p.list;
+            List<string> temp = new List<string>();
+            for (int i = 0; i < p.list.Count; i++)
+            {
+                if (i == 0)
+                    temp.Add("Welcome to Legend of Mir 2!");
```

Comment:
Please remove this fixed line addition (and the check above). I see no need to force a title when the rest of the text is driven by a text file.